code/modules/tgui_panel/external.dm

```diff
@@ -4,6 +4,7 @@
  */
 
 /client/var/datum/tgui_panel/tgui_panel
+/client/var/datum/tgui_modal/tgui_modal
```

Comment:
Create a tgui_modal module and try to keep as many procs and vars contained in it as possible, that includes all client procs and vars.